apollo-runtime/src/main/java/com/apollographql/android/impl/util/HttpException.java

```diff
@@ -1,15 +1,37 @@
 package com.apollographql.android.impl.util;
 
+import javax.annotation.Nullable;
+
 import okhttp3.Response;
 
 public class HttpException extends RuntimeException {
-  private final okhttp3.Response rawResponse;
+  private final int code;
+  private final String message;
+  private final Response rawResponse;
```

Comment:
Since `Response` isn't `Serializable`, this is a good time to add `transient` to this since technically `Exceptions` should be `Serializable`